app/assets/stylesheets/application.tailwind.css

```diff
@@ -1,6 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import "tooltip.css";
```

Comment:
Do we need to import this  ? any css file in here are automatically included by rails.,